totp/__init__.py

```diff
@@ -2,17 +2,48 @@
 #
 # Print a TOTP token getting the shared key from pass(1).
 
-import getpass
 import os
 import platform
 import re
 import subprocess
 import sys
-from base64 import b32decode
+import urllib.parse
 
 import onetimepass
 
 
+DIGITS_DEFAULT = 6
+
+
+class BackendError(Exception):
+    backend_name = '<none>'
+
+    def __init__(self, msg):
```

Comment:
What's the point of this? Looks pretty much like what `Exception` does by default; it takes `args` and keeps that in `exception.args`.

Reply:
My point was providing a more semantic name for the error message than `args[0]`, but I guess we don't need that extra complexity for such a simple case. Moreover the community seems to be ok with that default :-)